setup/install/providers/aws/aws-ec2.md

```diff
@@ -25,10 +25,13 @@ Use this option to deploy Spinnaker, if you are familar with deployment using [A
 1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
 2. Download [the template](https://d3079gxvs8ayeg.cloudfront.net/templates/managing.yaml) locally to your workstation.
 
-    2.a (Optional). Add additional managed account as shown on line 158 in the SpinnakerAssumeRolePolicy section of the downloaded template file.
-3. Creating the CloudFormation Stack
+    2.a Search for ‘SpinnakerInstanceProfileArn’ and comment out the line.
+    
+    2.b (Optional). Add additional managed account as shown in the `SpinnakerAssumeRolePolicy` section of the downloaded template file.
+3. Create the CloudFormation Stack:
     * __Create Stack__ > __Upload a template to Amazon S3__ > __Browse to template you downloaded in Step-2 above__ > __Next__
-    * Enter __Stack Name__ as spinnaker-**managing**-infrastructure-setup and follow the prompts on screen to create the stack
+    * Enter __Stack Name__ as spinnaker-**managing**-infrastructure-setup and follow the prompts on screen to create the stack.
+    * From the dropdown select **‘UseAccessKeyForAuthentication’** as True to get the Access and Secret Key.
```

Comment:
Does this also assume Spinnaker is deployed on EKS?

Reply:
For, managing accounts we need access and secret keys. In the CLI by default instruction is provided to keep it true, whereas in the Manual no instructions are provided for the Authentication purpose. Again this was validated on Spinnaker being deployed all Kubernetes platforms

Reply:
But this is not needed if Spinnaker is deployed directly to EC2 (not on Kubernetes).  Option 2 below more clearly states the options for the UseAccessKeyForAuthentication option.  So this should be something like:
If you want to use Access Keys and Secrets to run Spinnaker, set the UseAccessKeyForAuthentication parameter to True.  If you want to use an EC2 instance profile to run Spinnaker, set the UseAccessKeyForAuthentication parameter to False.

Reply:
For a localdebian spinnaker or distributed spinnaker if we need the halyard to access AWS to Bake, Create Instance or anything IAM roles and user access is mandatory. Which means Access and Secret are needed.

Reply:
The SpinnakerManaged role is required, but EC2 instance profile can be used by Spinnaker to assume the SpinnakerManager role in the case of localdebian install on EC2.  User access keys are not required in that case.